assets/scss/components.scss

```
@@ -0,0 +1,302 @@
+@media (max-width: 640px) {
+  .post-list { @apply grid-cols-1 gap-y-1; }
+  .post-list .post-date { @apply text-left text-sm; }
+  .post-list li, .post-list .post-row { @apply mb-3; }
+}
+
+/* Notes */
+
+/* Notes photo masonry — CSS columns, FOUC fade preserved. */
+.masonry-columns {
+  @apply w-full mx-auto p-0 opacity-0;
+  column-count: 3;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)